controller/backing_image_controller.go

```diff
@@ -486,7 +491,7 @@ func (bic *BackingImageController) handleBackingImageDataSource(bi *longhorn.Bac
 	}
 
 	// Check if the data source already finished the 1st file preparing.
-	if !bids.Spec.FileTransferred && bids.Status.CurrentState == longhorn.BackingImageStateReadyForTransfer {
+	if !bids.Spec.FileTransferred && bids.Status.CurrentState == longhorn.BackingImageStateReady {
```

Comment:
Curious. Why chage it to BackingImageStateReady instead of BackingImageStateReadyForTransfer?

Reply:
Right now the BackingImageDataSource will update the state to ready by itself when the BIM takes over the file:
https://github.com/shuo-wu/backing-image-manager/blob/6dc15c6ec60a126590ac117214122c1c1a15e5cd/pkg/manager/service.go#L401
https://github.com/shuo-wu/backing-image-manager/blob/6dc15c6ec60a126590ac117214122c1c1a15e5cd/pkg/datasource/service.go#L318-L319

Reply:
I see. Thank you.